WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift

```diff
@@ -8,6 +8,7 @@ import WordPressShared
 import MobileCoreServices
 import WordPressEditor
 import AVKit
+import MobileCoreServices
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/wordpress-mobile/WordPress-iOS/blob/104b0c52a1168c8bc73da5fee16d942972ba7c48/WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift#L11"></span>Imports should be unique<br />
<code>duplicate_imports</code> <code>AztecPostViewController.swift:11</code></td>
    </tr>
  </tbody>
</table>